src/Microsoft.ML.Experimental/Experimental.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Experimental
+{
+    public static class Experimental
```

Comment:
>Experimental [](start = 24, length = 12)

Rname to MLContextExtensions. Let's not use up the "experimental" class name. We might have several experimental features.